recipes/trino/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.305.0" %}
+
+package:
+  name: trino-python-client
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/t/trino/trino-{{ version }}.tar.gz
+  sha256: 795d3b1a1bd0373ebf6af9a87ecc2aa6f4aa32e3098a8e05677d528a5324c098
+
+build:
+  # noarch: python
```

Comment:
Why not?

Reply:
Good point. I tend to want to try the different versions in CI explicitly before I switch on `noarch`

Reply:
Done, PTAL :)